src/Allegro.DotnetSdk/Sdk/Allegro.NET.Sdk.CSharp.targets

```diff
@@ -3,21 +3,14 @@
   <!-- Setup this SDKs defaults -->
   <PropertyGroup>
     <!-- Enable default analyzers if CPM is not used. -->
-    <UseAllegroDotnetSdkDefaultAnalyzers Condition="'$(UseAllegroDotnetSdkDefaultAnalyzers)' == '' And '$(ManagePackageVersionsCentrally)' != 'true'">true</UseAllegroDotnetSdkDefaultAnalyzers>
     <AllegroDotnetSdkEnableStyleCopConfig Condition="'$(AllegroDotnetSdkEnableStyleCopConfig)' == ''">true</AllegroDotnetSdkEnableStyleCopConfig>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(UseAllegroDotnetSdkDefaultAnalyzers)' == 'true'">
-    <AllegroDotnetSdkEnableAsyncFixer Condition="'$(AllegroDotnetSdkEnableAsyncFixer)' == ''">true</AllegroDotnetSdkEnableAsyncFixer>
-    <AllegroDotnetSdkEnableMeziantou Condition="'$(AllegroDotnetSdkEnableMeziantou)' == ''">true</AllegroDotnetSdkEnableMeziantou>
-    <AllegroDotnetSdkEnableStyleCop Condition="'$(AllegroDotnetSdkEnableStyleCop)' == ''">true</AllegroDotnetSdkEnableStyleCop>
-  </PropertyGroup>
-
   <!-- 
     Include stylecop configuration if we include StyleCop analyzer within this Sdk,
     or if it was included externally as GlobalPackageReference when CPM is enabled.
   -->
-  <ItemGroup Condition="'$(AllegroDotnetSdkEnableStyleCopConfig)' == 'true' And ('$(AllegroDotnetSdkEnableStyleCop)' == 'true' Or @(GlobalPackageReference->WithMetadataValue('Identity','StyleCop.Analyzers')->Count()) > 0)">
+  <ItemGroup Condition="'$(AllegroDotnetSdkEnableStyleCopConfig)' == 'true' Or @(GlobalPackageReference->WithMetadataValue('Identity','StyleCop.Analyzers')->Count()) > 0)">
```

Comment:
Mam nadzieję że nie usunąłem za dużo

Reply:
Chyba spoko. Zastanawiam się czy nie dorobić podobnego warunku jak na GlobalPackageReference, na PackageReference, żeby obsłużyć projekty które "po staremu" mają.

Z drugiej strony, GlobalPackageReference jest niezależne od użycia CPMa i idealne dla sln-wide analizatorów, więc można to uznać za best-practice tak czy siak i wspierać tylko GPR tak jak teraz. I wtedy zmienić notkę w readme że zalecane jest referencjonowanie analizatorów przez GPR z cpmem i bez.

Reply:
Zalecenie jest pokazane w readme zgodnie z rekomendacją tylko na GRP z CPM. Zostawiam jak jest